enatega-multivendor-app/src/screens/ItemDetail/ItemDetail.js

```
@@ -24,46 +18,28 @@ import useNetworkStatus from '../../utils/useNetworkStatus'
 const TOP_BAR_HEIGHT = height * 0.08
-const HEADER_MAX_HEIGHT = height * 0.40
-const HEADER_MIN_HEIGHT = height * 0.05 + TOP_BAR_HEIGHT
-const SCROLL_RANGE = HEADER_MAX_HEIGHT - HEADER_MIN_HEIGHT
+const HEADER_MAX_HEIGHT = height * 0.4
+const HEADER_MIN_HEIGHT = TOP_BAR_HEIGHT
+const SCROLL_RANGE = HEADER_MAX_HEIGHT
+
 function ItemDetail(props) {
-  // Analytics
-  const Analytics = analytics()
   const { food, addons, options, restaurant } = props?.route?.params
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)